test/verify/check-machines

```diff
@@ -116,6 +116,10 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
             m.execute("until test -e /run/libvirt/libvirt-sock; do sleep 1; done")
             m.execute("chmod o+rwx /run/libvirt/libvirt-sock")
 
+    def toggleVmRow(self, vmName):
+        b = self.browser
+        b.click("tbody tr[data-row-id=vm-{0}] th".format(vmName)) # click on the row header
```

Comment:
Nitpick: defining the `b` alias just for one usage is unnecessary.

Reply:
Done